Samples/Logging with Stackexchange Exceptional (WebLog)/Resources/Litium.Community.ExceptionalWebLog/obj/Debug/net472/Litium.Community.ExceptionalWebLog.AssemblyInfo.cs

```diff
@@ -0,0 +1,24 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Reflection;
+
+[assembly: System.Reflection.AssemblyCompanyAttribute("Litium.Community.ExceptionalWebLog")]
+[assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
+[assembly: System.Reflection.AssemblyDescriptionAttribute("WebLog implementation of exceptional for Litium.Owin.")]
+[assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
+[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0-pre01")]
+[assembly: System.Reflection.AssemblyProductAttribute("Litium.Community.ExceptionalWebLog")]
+[assembly: System.Reflection.AssemblyTitleAttribute("Litium.Community.ExceptionalWebLog")]
+[assembly: System.Reflection.AssemblyVersionAttribute("1.0.0.0")]
+
+// Generated by the MSBuild WriteCodeFragment class.
+
```

Comment:
Is exceptional needed for this sample?

Reply:
thanks, VS Code added them for no reason